infrastructure/prime-app-template.yml

```diff
@@ -1246,3 +1252,156 @@ objects:
+                image: >-
+                  image-registry.openshift-image-registry.svc:5000/openshift/python:3.8
+                command:
+                - bash
+                - '-c'
+                - >-
+                  echo -e "-------- STARTING CRON --------\n"
+
+                  echo -e "-------- Getting access_token --------\n"
+
+                  TOKEN=$(curl -X POST
+                  "${KEYCLOAK_URL}/realms/${KEYCLOAK_REALM}/protocol/openid-connect/token"
+                  \
+                    -H "Content-Type: application/x-www-form-urlencoded" \
+                    -d "grant_type=client_credentials" \
+                    -d "client_id=${KEYCLOAK_CLIENT_ID}" \
+                    -d "client_secret=${KEYCLOAK_CLIENT_SECRET}" | python -c "import sys, json; print(json.load(sys.stdin)['access_token'])")
+
+                  echo -e ${TOKEN}
+
+                  echo -e "-------- Calling PRIME API --------\n"
+
+                  curl -v -X POST
+                  http://${SVC_NAME}-webapi:8080/api/Emails/management/statuses
+                  \
+                    -H "Authorization: Bearer ${TOKEN}" \
+                    -H "Content-Length: 0"
+
+                  echo -e "-------- CRON COMPLETE --------\n"
+                env:
+                  - name: KEYCLOAK_CLIENT_SECRET
+                    valueFrom:
+                      secretKeyRef:
+                        key: prime_service_account_client_secret
+                        name: prime-service-account
+                  - name: KEYCLOAK_CLIENT_ID
+                    valueFrom:
+                      secretKeyRef:
+                        key: prime_service_account_client_id
+                        name: prime-service-account
+                  - name: KEYCLOAK_URL
+                    valueFrom:
+                      secretKeyRef:
+                        key: KEYCLOAK_URL
+                        name: keycloak
+                  - name: KEYCLOAK_REALM
+                    valueFrom:
+                      secretKeyRef:
+                        key: KEYCLOAK_REALM
+                        name: keycloak
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 2Gi
+                  requests:
+                    cpu: 100m
+                    memory: 512Mi
+            restartPolicy: Never
+    schedule: '${CRON_SCHEDULE_STATUSES}'
+
+#   Template for job that activates api to send renewal emails 
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: ${SVC_NAME}-send-renewal-emails
+    labels:
+      app.kubernetes.io/part-of: ${SVC_NAME}
+  spec:
+    concurrencyPolicy: Forbid
+    jobTemplate:
+      spec:
+        template:
+          spec:
+            containers:
+              - name: ${SVC_NAME}-send-renewal-emails
+                image: >-
+                  image-registry.openshift-image-registry.svc:5000/openshift/python:3.8
```

Comment:
why python? Wouldn't alpine be enough?

Reply:
I'll try switching it to an alpine image I made for other cron job with python in it